python/paddle/device.py

```diff
@@ -13,10 +13,25 @@
 # limitations under the License.
 
 # TODO: define the functions to manipulate devices 
-# __all__ = ['cpu_places',
-#            'CPUPlace',
-#            'cuda_pinned_places',
-#            'cuda_places',
-#            'CUDAPinnedPlace',
-#            'CUDAPlace',
-#            'is_compiled_with_cuda']
+from paddle.fluid import core
+__all__ = [
+    'get_cudnn_version',
+    #            'cpu_places',
+    #            'CPUPlace',
+    #            'cuda_pinned_places',
+    #            'cuda_places',
+    #            'CUDAPinnedPlace',
+    #            'CUDAPlace',
+    #            'is_compiled_with_cuda'
+]
+
+
+def get_cudnn_version():
```

Comment:
the return value should be cached, as it will never change and this function may be called on each step for each OP that use it in dygraph mode.

Reply:
Done